Tools/setup/ubuntu.sh

```diff
@@ -62,17 +59,21 @@ sudo apt-get update -y --quiet
 sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends install \
 	astyle \
 	build-essential \
+	ca-certificates \
 	cmake \
 	cppcheck \
 	file \
 	g++ \
 	gcc \
 	gdb \
 	git \
+	gnupg \
+	gosu \
```

Comment:
Why here? We should try to keep actual PX4 dev dependencies straight and handle misc extras for containers separately.